tests/prediction/gradient_boosting_test.py

```diff
@@ -0,0 +1,58 @@
+import numpy as np
+import pytest
+from sklearn.datasets import load_iris
+from sklearn.ensemble import GradientBoostingClassifier, GradientBoostingRegressor
+
+from eis_toolkit import exceptions
+from eis_toolkit.prediction.gradient_boosting import (
+    gradient_boosting_classifier_predict,
+    gradient_boosting_classifier_train,
+    gradient_boosting_regressor_predict,
+    gradient_boosting_regressor_train,
+)
+
+X, y = load_iris(return_X_y=True)
```

Comment:
I would use upper case for these variables to highlight them in the test functions e.g.

~~~python
IRIS_X, IRIS_Y = ...
~~~